src/Sulu/Bundle/AdminBundle/Serializer/Subscriber/SaveWithFormDialogToolbarActionSubscriber.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Serializer\Subscriber;
+
+use JMS\Serializer\EventDispatcher\Events;
+use JMS\Serializer\EventDispatcher\EventSubscriberInterface;
+use JMS\Serializer\EventDispatcher\ObjectEvent;
+use Sulu\Bundle\AdminBundle\Admin\View\SaveWithFormDialogToolbarAction;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class SaveWithFormDialogToolbarActionSubscriber implements EventSubscriberInterface
```

Comment:
I think it would be good if we have a functional test case for this e.g.:

```php
$this->assertSame($expectedJson, $this->jmsSerializer->serialize($toolbarAction);
```